src/nft/helpers.ts

```diff
@@ -49,30 +54,99 @@ export const nftsFromOperations = (ops: Operation[]): NFT[] => {
 };
 
 export const nftsByCollections = (
-  nfts: NFT[] = [],
+  nfts: Array<ProtoNFT | NFT> = [],
   collectionAddress?: string
-): CollectionWithNFT[] => {
-  const filteredNfts = collectionAddress
-    ? nfts.filter((n) => n.collection.contract === collectionAddress)
-    : nfts;
+): Record<string, Array<ProtoNFT | NFT>> | Array<ProtoNFT | NFT> => {
```

Comment:
This return type is a bit hard to read, maybe add a small comment on this function to explain what it does.

Reply:
Done 👌